ui/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "dependencies": {},
+  "devDependencies": {
+    "grunt": "^0.4.5",
+    "grunt-autoprefixer": "^2.0.0",
+    "grunt-contrib-compass": "^1.0.0",
+    "grunt-contrib-uglify": "^0.7.0",
+    "grunt-contrib-copy": "^0.7.0",
+    "load-grunt-tasks": "^3.1.0",
+    "time-grunt": "^1.0.0"
+  },
+  "engines": {
+    "node": ">=0.10.0"
+  },
+  "repository" : {
+    "type" : "git",
+    "url" : "https://github.com/vmware/docker-volume-vsphere.git"
+  },
+  "author": {
+    "name": "Mark Soper",
```

Comment:
We generally avoid individual's name in code/repository. I would suggest to replace with following;
name = SABU, VMware
email = cna-storage@vmware.com
url = https://github.com/vmware/docker-volume-vsphere